website/siteConfig.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+/* List of projects/orgs using your project for the users page */
+const users = [
+    /*
+    {
+        caption: 'User1',
+        image: '/test-site/img/docusaurus.svg',
+        infoLink: 'https://www.facebook.com',
+        pinned: true,
+    },
+    */
+];
+
+const siteConfig = {
+    title: 'BlueWhale',
+    tagline: 'Large-Scale Reinforcement Learning Engine',
+    url: 'https://github.com/facebookresearch/BlueWhale',
+    baseUrl: '/site/',
+    projectName: 'bluewhale',
```

Comment:
This might be ok, but you might want to case it like the GitHub project - `BlueWhale`.

Reply:
Actually no - this is case sensitive. So you need `BlueWhale` after all.